datalad/distributed/create_sibling_ria.py

```diff
@@ -636,6 +644,12 @@ def _create_sibling_ria(
         if group:
             # TODO; do we need a cwd here?
             subprocess.run(chgrp_cmd, cwd=quote_cmdlinearg(ds.path))
+        # finally update server
+        if post_update_hook:
+            # Conditional on post_update_hook, since one w/o the other doesn't
+            # seem to make much sense.
+            subprocess.run("git update-server-info",
+                           cwd=quote_cmdlinearg(ds.path))
```

Comment:
I don't think the value for `cwd` should be quoted.

(Also, I suppose `call_git` could be used here?)

Reply:
Yes. I just thought for the bugfix I'll keep it consistent within the rest of the implementation. The command itself will have to go through some refactoring anyway. Then those kind of changes can be made everywhere.

Reply:
^ That is re `call_git`.

Re quotation:
Really? What about spaces in the path?

Reply:
Well - screw it. Can as well go in here. Pushed ;-)

Reply:
> Yes. I just thought for the bugfix I'll keep it consistent within the rest of the implementation.

Sure.

> You're right. However, rationale of doing the same thing as in other calls next to it remains.

Yep, my comment was only about the `cwd` argument for this `subprocess.run` call.